moveit_core/planning_request_adapter/include/moveit/planning_request_adapter/planning_request_adapter.h

```diff
@@ -63,6 +63,15 @@ class PlanningRequestAdapter
   {
   }
 
+  /// Initialize parameters using the passed NodeHandle
+  // TODO - Make initialize() a pure virtual function
+  virtual void initialize(const ros::NodeHandle& node_handle)
+  {
+    ROS_WARN_NAMED("planning_request_adapter",
+                   "Function initialize() is not implemented. All parameters should be"
```

Comment:
```suggestion
                   "Implementation of function initialize() is missing, thus preventing to load parameters from the passed NodeHandle / namespace."
```

Reply:
Isn't this a bit confusing since not all adapters load parameters?